conf/application.intg.conf

```diff
@@ -15,6 +15,8 @@ consignmentapi.domain="https://api.tdr-integration.nationalarchives.gov.uk"
 
 environment=${ENVIRONMENT}
 
+logout.url="https://tdr-integration.nationalarchives.gov.uk/sign-out"
```

Comment:
Is the base URL already an environment variable or a config value so we don't need to set this in each config file?

Reply:
No, we've got specific variables in each file for auth.callback as well. There's nothing I can see in the base config I can see that we can re-use here. There aren't any environment variables in the ECS task for the base url either. We could do it but that's a bigger change with terraform changes which would be better on another ticket if you're going to do it.